From 5b6e47a7b1eb8c286e7c40479ba19131a2f696d7 Mon Sep 17 00:00:00 2001
From: "Sar Ashki, Babak" <Babak.SarAshki@windriver.com>
Date: Thu, 26 Dec 2019 12:00:31 -0800
Subject: [PATCH] workaround: Adjust keystone naming to poky

Issue 43:
In the puppet apply, openstack-keystone is used as the keystone package
name, but in yocto, the package name is 'keystone'.
---
 manifests/params.pp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/manifests/params.pp b/manifests/params.pp
index 0d61acfb..b2bc44b1 100644
--- a/manifests/params.pp
+++ b/manifests/params.pp
@@ -18,7 +18,7 @@ class keystone::params {
       $openidc_package_name         = 'libapache2-mod-auth-openidc'
     }
     'RedHat': {
-      $package_name                 = 'openstack-keystone'
+      $package_name                 = 'keystone'
       $service_name                 = 'openstack-keystone'
       $keystone_wsgi_script_path    = '/var/www/cgi-bin/keystone'
       $python_memcache_package_name = 'python-memcached'
-- 
2.23.0

